federation/cmd/federation-controller-manager/app/options/options.go

```diff
@@ -115,6 +120,7 @@ func (s *CMServer) AddFlags(fs *pflag.FlagSet) {
 	fs.StringVar(&s.ServiceDnsSuffix, "service-dns-suffix", s.ServiceDnsSuffix, "DNS Suffix to use when publishing federated service names.  Defaults to zone-name")
 	fs.IntVar(&s.ConcurrentServiceSyncs, "concurrent-service-syncs", s.ConcurrentServiceSyncs, "The number of service syncing operations that will be done concurrently. Larger number = faster endpoint updating, but more CPU (and network) load")
 	fs.IntVar(&s.ConcurrentReplicaSetSyncs, "concurrent-replicaset-syncs", s.ConcurrentReplicaSetSyncs, "The number of ReplicaSets syncing operations that will be done concurrently. Larger number = faster endpoint updating, but more CPU (and network) load")
+	fs.IntVar(&s.ConcurrentDeploymentSyncs, "concurrent-deployment-syncs", s.ConcurrentDeploymentSyncs, "The number of Deployments syncing operations that will be done concurrently. Larger number = faster endpoint updating, but more CPU (and network) load")
```

Comment:
But here we did not share ConcurrentReplicaSetSyncs
https://github.com/kubernetes/kubernetes/blob/fc6eb52d0503c8353596a0b8d2b393fe7ed0041c/cmd/kube-controller-manager/app/options/options.go#L131